src/metabase/routes.clj

```diff
@@ -13,6 +14,14 @@
             [ring.util.response :as resp]
             [stencil.core :as stencil]))
 
+(defn- base-href []
+  (str (.getPath (clojure.java.io/as-url (public-settings/site-url))) "/"))
+
+(defn- escape-script [text]
```

Comment:
Probably makes sense to clarify this a bit and call it `escape-script-tags` or at the very least `escape-scripts`

Also in this case it's idiomatic Clojure to use `s` if you have a generic string parameter, so I would tweak this to be

```clojure
(defn- escape-script-tags [s]
  ...)
```